### ISSUE TITLE
Change to proper slack application

### ISSUE THREAD

**Issue by alexjpaz** [alexjpaz/prathu#42]
Right now prathu acts as a "bot" in slack. This seems to be deprecated in favor of "apps" in slack.

**Comment by ajmath** [comment]
This would mean that he could be added to any organization right?

**Comment by alexjpaz** [comment]
Yes – That would be my goal. Although there probably is more work to make him "generic" (e.g. actually using a brain backend with per organization values).
